_includes/live-demos/drive-standalone-browse-controls/index.js

```diff
@@ -0,0 +1,20 @@
+(function () {
+  var button = document.querySelector('.tinydrive-standalone-demo-browse');
+
+  button.addEventListener('click', function () {
+    var controls = tinydrive.controls.browse({
+      demo_files_url: '{{ site.baseurl }}/demo/tiny-drive-demo/demo_files.json',
+      token_provider: function (success) { success({ token: 'fake-token' }); },
```

Comment:
No single line functions please. Can you please break this up over multiple lines to improve readability.